File: promocao.php
<?php
echo ("Qual seu nome ? ");
$nome = (string) trim(fgets(STDIN));
echo ("Qual seu sexo ? [F/M] ");
$sexo = (string) trim(fgets(STDIN));
echo ("Qual valor de sua compra ? ");
$valor = (float) trim(fgets(STDIN));

if ($sexo == "F" ||$sexo == "f") {
    $desconto = ($valor * 10) / 100;
    $novoValor = $valor - $desconto;
} elseif ($sexo == "M" ||$sexo == "m") {
    $desconto = $valor * (5 / 100);
    $novoValor = $valor - $desconto;
} else {
    echo ("Resposta inválida");
}

if ($sexo == "F" ||$sexo == "f") {
    echo ("Olá, " . $nome . " vc ganhou um desconto de 10%\n");
    echo ("O valor de " . $valor . " se tornou " . $novoValor. " um desconto de " . $desconto . " reais");
} elseif ($sexo == "M" ||$sexo == "m") {
    echo ("Olá, " . $nome . " vc ganhou um desconto de 5%\n");
    echo ("O valor de " . $valor . " se tornou " . $novoValor. " um desconto de " . $desconto . " reais");
} else {
    die();
}
?>

File: promocao.py
nome = input("Qual seu nome? ")
sexo = input("Qual seu sexo? [F/M]")
valor = int(input("Qual o valor de sua compra? "))

if sexo == "F" or sexo == "f":
    desconto = (valor * 10) / 100
    novoValor = valor - desconto
elif sexo == "M" or sexo == "m":
    desconto = valor * (5 / 100)
    novoValor = valor - desconto
else: print("Resposta inválida")

if sexo == "F" or sexo == "f":
    print("Olá, ", nome, " vc ganhou um desconto de 10%")
    print("O valor de sua compra era", valor, "agora é", novoValor, "reais")
elif sexo == "M" or sexo == "m":
    print("Olá, ", nome, " vc ganhou um desconto de 5%")
    print("O valor de sua compra era", valor, "agora é", novoValor, "reais")
